dev-lang/spidermonkey/spidermonkey-1.8.5.recipe

```diff
@@ -34,19 +36,29 @@ BUILD_REQUIRES="
 BUILD_PREREQUIRES="
 	cmd:gcc$secondaryArchSuffix
 	cmd:ld$secondaryArchSuffix
+	cmd:libtoolize
```

Comment:
```diff
-	cmd:libtoolize
+	cmd:libtoolize$secondaryArchSuffix
```

Reply:
`libtoolize` should actually be architecture-independent (it's a bash script). But since it's packaged with libltdl...